ThirdParty/npm/zip.js

```diff
@@ -0,0 +1,8 @@
+import * as zip from "@zip.js/zip.js";
+// zip.configure({
```

Comment:
I couldn't figure out how to link to the `pako` files here, but it should be possible to use zip.js with the `pako` files that we already install with npm per the [`@zip.js/zip.js` docs](https://gildas-lormeau.github.io/zip.js/core-api.html#configuration).

Reply:
First thing I would check is the paths

> The first URL is relative to the base URI of the document. The other URLs are relative to the URL of the first script. 

But do we actually need zip.js to use the pako files? zip.js runs its own workers by default. I assume that this is what was already happening but I could be wrong or misremembering from when we talked with @mramato.

Reply:
I wonder if the original files we have in Workers/ in `main` are from an old version of zip-js? Looking at the source of https://github.com/gildas-lormeau/zip.js/releases/tag/v2.0.0 it looks like that's the case.

Reply:
Can `deflate.js` and `inflate.js` be removed then? Also can the commented code here be removed?

Reply:
From slack, I think it's fine to remove them. The default behavior of the zip.js package is to use web workers, and in Sandcastle/tests it seems to work fine without any additional config options. I removed them in 7b40a60.